ray-operator/config/samples/ray-service.tpu-multi-host.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: s-d-tpu
+spec:
+  serveConfigV2: |
+    applications:
+      - name: stable_diffusion
+        import_path: stable_diffusion.stable_diffusion_tpu:deployment
+        runtime_env:
+          working_dir: "https://github.com/ryanaoleary/serve_config_examples/archive/refs/heads/tpu-example.zip"
+          pip:
+            - pydantic<2
+            - google-api-python-client
+            - pillow
+            - diffusers==0.7.2
+            - transformers==4.24.0
+            - flax
+            - ml_dtypes==0.2.0
+            - jax[tpu]==0.4.11
+            - -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
+            - fastapi
+            - pillow
+            - ray[serve,default]==2.9.0
+  rayClusterConfig:
+    rayVersion: '2.9.0' # Should match the Ray version in the image of the containers
+    ######################headGroupSpecs#################################
+    # Ray head pod template.
+    headGroupSpec:
+      # The `rayStartParams` are used to configure the `ray start` command.
+      # See https://github.com/ray-project/kuberay/blob/master/docs/guidance/rayStartParams.md for the default settings of `rayStartParams` in KubeRay.
+      # See https://docs.ray.io/en/latest/cluster/cli.html#ray-start for all available options in `rayStartParams`.
+      rayStartParams:
```

Comment:
`rayStartParams: {}`

Reply:
Done in [3b44939](https://github.com/ray-project/kuberay/pull/2198/commits/3b449390808f87876c499aacb31dac657b903191).